lms/djangoapps/ccx/views.py

```diff
@@ -410,7 +410,7 @@ def ccx_invite(request, course, ccx=None):
     action = request.POST.get('enrollment-button')
     identifiers_raw = request.POST.get('student-ids')
     identifiers = _split_input_list(identifiers_raw)
-    auto_enroll = True if 'auto-enroll' in request.POST else False
+    auto_enroll = True
```

Comment:
No, By default test case only support student who are member of class(enrolled one )
see https://github.com/mitocw/edx-platform/blob/enhancement/aq/disable_auto_enrol_ccx/lms/djangoapps/ccx/tests/test_views.py#L539